### ISSUE TITLE
Remove twisted dependency from "bart-registrant"

### ISSUE THREAD

**Issue by kmjonsson** [sgas/bart#40]
Twisted depends on a lot of modules and for the simple task of sending the XML files to luts it feels a little bit overkill for the task.

Twisted are also not available for python3 in CentOS 7+

**Comment by edelmann** [comment]
Yeah.  I'll see if I can un-twist bart-registrant within the next few days.
